src/linked-list.js

```diff
@@ -4,6 +4,40 @@ class LinkedList {
     this.tail = null;
     // Do not modify anything inside of the constructor
   }
+  addToTail(input) {
+    const node = {
+      value: input,
+      next: null
+    };
+    if (this.head === null) {
+      this.head = node;
+      this.tail = node;
+    } else {
+      this.tail.next = node;
+      this.tail = node;
+    }
+  }
+  removeHead() {
+    const headObj = this.head.value;
+    if (this.head !== null) {
+      this.head = this.head.next;
+    }
+    return headObj;
+  }
+  contains(input) {
+    const cb = (node) => {
```

Comment:
Nice use of a helper function here. This is very nitpicky, but a callback function is defined as a function that is passed to another function as an argument, and that's not strictly what you are doing with this helper function. Just a slightly misleading name for this function that I wanted to clear up.